impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -0,0 +1,273 @@
+            renderer.GetContext()->GetSamplerLibrary()->GetSampler(
+                sampler_descriptor));
+
+        pass.AddCommand(std::move(cmd));
+
+        return true;
+      };
+  std::shared_ptr<Texture> out_texture = renderer.MakeSubpass(
+      "Gaussian Blur Filter", subpass_size, subpass_callback);
+  return out_texture;
+}
+
+std::shared_ptr<Texture> MakeBlurSubpass(
+    const ContentContext& renderer,
+    std::shared_ptr<Texture> input_texture,
+    const SamplerDescriptor& sampler_descriptor,
+    const GaussianBlurFragmentShader::BlurInfo& blur_info) {
+  // TODO(gaaclarke): This doesn't render if the blur_info.sigma == 0.
+  ISize subpass_size = input_texture->GetSize();
+  ContentContext::SubpassCallback subpass_callback =
+      [&](const ContentContext& renderer, RenderPass& pass) {
+        GaussianBlurVertexShader::FrameInfo frame_info{
+            .mvp = Matrix::MakeOrthographic(ISize(1, 1)),
+            .texture_sampler_y_coord_scale = 1.0};
+
+        HostBuffer& host_buffer = pass.GetTransientsBuffer();
+
+        Command cmd;
+        ContentContextOptions options = OptionsFromPass(pass);
+        options.primitive_type = PrimitiveType::kTriangleStrip;
+        cmd.pipeline = renderer.GetGaussianBlurPipeline(options);
+        BindVertices<GaussianBlurVertexShader>(cmd, host_buffer,
+                                               {
+                                                   {Point(0, 0), Point(0, 0)},
+                                                   {Point(1, 0), Point(1, 0)},
+                                                   {Point(0, 1), Point(0, 1)},
+                                                   {Point(1, 1), Point(1, 1)},
+                                               });
+
+        GaussianBlurFragmentShader::BindTextureSampler(
+            cmd, input_texture,
+            renderer.GetContext()->GetSamplerLibrary()->GetSampler(
+                sampler_descriptor));
+        GaussianBlurVertexShader::BindFrameInfo(
+            cmd, host_buffer.EmplaceUniform(frame_info));
+        GaussianBlurFragmentShader::BindBlurInfo(
+            cmd, host_buffer.EmplaceUniform(blur_info));
+        pass.AddCommand(std::move(cmd));
+
+        return true;
+      };
+  std::shared_ptr<Texture> out_texture = renderer.MakeSubpass(
+      "Gaussian Blur Filter", subpass_size, subpass_callback);
+  return out_texture;
+}
+
+/// Given a desired |scalar|, will return the scalar that gets close but leaves
+/// |size| in integer sizes.
+Vector2 CalculateIntegerScale(Scalar scalar, ISize size) {
+  ISize new_size(size.width / scalar, size.height / scalar);
+  return Vector2(size.width / static_cast<Scalar>(new_size.width),
+                 size.height / static_cast<Scalar>(new_size.height));
+}
+
+/// Calculate how much to scale down the texture depending on the blur radius.
+/// This curve was taken from |DirectionalGaussianBlurFilterContents|.
+Scalar CalculateScale(Scalar radius) {
```

Comment:
If I wanted to change this scaling, is this where I would do it?

As a follow up we may align to skia's curve, which scales sigma down to 4 proportionally.

Reply:
Yep, oh interesting.  This was me copying the one from the other blur.  I think we'll have to revisit this since I think it's a bit too aggressive (because of the artifacts when animating the sigma).